### ISSUE TITLE
change of learning rate does not work

### ISSUE THREAD

**Issue by zaoyehuo** [fchollet/keras#4497]
Please make sure that the boxes below are checked before you submit your issue. Thank you!

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [x] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

optimizers.rmsprop(lr=0.001, rho=0.9, epsilon=1e-06)
model.compile(loss='mse', optimizer='rmsprop')

I change the lr=0.001 to 0.1,10,etc  but I got the same loss.

And I tried to change many things in my LSTM,including batch_size,nb_epoch,output_dim,optimizer...all the things I can think to change, but I just got similar loss.What can I do now?

**Comment by jmhessel** [comment]
If you use `optimizer = 'rmsprop'` you're going to get an rmsprop with the default settings. what you want is

```
my_rmsprop = RMSprop(...)
model.compile(... , optimizer =  my_rmsprop)
```

Also, this might be a question better suited for the [google group](https://groups.google.com/forum/#!forum/keras-users) because it's not really a bug.

**Comment by MoyanZitto** [comment]
@zaoyehuo 
@jmhessel is correct, you can pass either a str or an optimizer object to the argument “optimizer”, if a str is provided, Keras will use the predefined optimizer indicated by the str, you can find them in keras.optimizers. If an optimizer object is provided, keras will use it.
